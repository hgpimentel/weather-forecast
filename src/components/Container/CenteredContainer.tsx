import styled from "styled-components";

const CenteredContainer = styled.div`
  display: flex;
  justify-content: center;
`;

export default CenteredContainer;
