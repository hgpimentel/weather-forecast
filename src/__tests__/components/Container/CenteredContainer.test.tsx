import renderer from "react-test-renderer";
import CenteredContainer from "../../../components/Container/CenteredContainer";

describe("Centered container component", () => {
  it("renders correctly", () => {
    const component = renderer.create(<CenteredContainer />).toJSON();

    expect(component).toMatchSnapshot();
  });
});
