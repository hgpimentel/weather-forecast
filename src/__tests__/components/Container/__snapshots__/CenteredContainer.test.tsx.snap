// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Centered container component renders correctly 1`] = `
<div
  className="sc-bdnxRM jNerll"
/>
`;
